src/Hedgehog/Script.fsx

```diff
@@ -57,8 +58,8 @@ Gen.printSample genLeapYear
 
 // Fails due to integer overflow
 Property.print <| property {
-    let! x = Gen.int
-    let! y = Gen.int
+    let! x = Gen.int <| Range.constantBounded ()
+    let! y = Gen.int <| Range.constantBounded ()
```

Comment:
Yeah, I was also going back and forth on that one. Let's make `Gen` similar to Haskell and we may come back to this one later.